src/pages/BusinessSpecifics/BusinessSpecifics.tsx

```diff
@@ -2,14 +2,35 @@ import { FormControl, TextField, Button, MenuItem, Select } from "@mui/material"
 import { useState } from "react"
 import { currencies } from "./currencies"
 
-export const BusinessSpecifics = () => {
 
-    const [BusinessName, setBusinessName] = useState("")
-    const [LocationName, setLocationName] = useState("")
+export const BusinessSpecifics = () => {
+    const url = 'https://invent0ry-back3nd.herokuapp.com/businesses/'
+    const [businessName, setBusinessName] = useState("")
+    const [locationName, setLocationName] = useState("")
     const [currency, setCurrency] = useState(currencies.USD)
 
     const currencyList = Object.keys(currencies).map(key => (<MenuItem value={key}>{key}</MenuItem>))
 
+    function handleSubmit(){
+        const options = {
+            method: "POST",
+            headers: {
+              Accept: "application/json",
+              "Content-Type": "application/json;charset=UTF-8",
+            },
+            body: JSON.stringify({
+                'name': businessName,
+                'location_name': locationName,
+                'currency': currency
+            })}
+            fetch(url, options)

```

Comment:
Instead of using this native solution for requests we can use the "axios" package that is installed

Reply:
I did end up using axios for the built in stringify and responses, but scratched both altogether for amplify